run/opencl/opencl_device_info.h

```diff
@@ -32,12 +32,11 @@
 #define DEV_NV_C35                  (1 << 17)   //131072
 #define DEV_NV_MAXWELL              (1 << 18)   //262144
 #define DEV_NV_MAXWELL_PLUS         (1 << 19)   //524288
-#define DEV_NV_PASCAL               (1 << 20)   //1M
-#define DEV_USE_LOCAL               (1 << 21)   //2M
-#define DEV_NO_BYTE_ADDRESSABLE     (1 << 22)   //4M
-#define PLATFORM_MESA               (1 << 23)   //8M
-#define PLATFORM_BEIGNET            (1 << 24)   //16M
-#define PLATFORM_POCL               (1 << 25)   //32M
+#define DEV_USE_LOCAL               (1 << 20)   //1M
+#define DEV_NO_BYTE_ADDRESSABLE     (1 << 21)   //2M
+#define PLATFORM_MESA               (1 << 22)   //4M
+#define PLATFORM_BEIGNET            (1 << 23)   //8M
+#define PLATFORM_POCL               (1 << 24)   //16M
```

Comment:
I think kernels aren't automatically properly rebuilt when a dependency like `opencl/opencl_device_info.h` is updated. For this reason, we might want to avoid renumbering of flags here - just skip the flag bit that used to be for Pascal.